docs/conceptual/performance-model.rst

```diff
@@ -24,9 +160,9 @@ to use ROCm Compute Profiler to optimize your code.
 
 .. note::
 
-   In this chapter, **MI2XX** refers to any of the CDNA2 architecture-based AMD
+   In this documentation, **MI2XX** refers to any of the CDNA2 architecture-based MI200 series accelerators such as AMD
    Instinct MI250X, MI250, and MI210 accelerators interchangeably in cases
-   where the exact product at hand is not relevant.
+   where the exact product at hand is not relevant. Likewise **MI3XX** refers to any of the CDNA3 or CDNA4 architecture-based  MI300 series accelerators such as AMD Instinct MI300A, MI300X, MI325X, MI350X, and MI355X.
 
    For a comparison of AMD Instinct accelerator specifications, refer to
    :doc:`Hardware specifications <rocm:reference/gpu-arch-specs>`. For product
```

Comment:
For product  details, see the https://www.amd.com/en/products/accelerators/instinct.html.
Here, we don't have to go to the details for each arch any more